google/cloud/storage/transfer_manager.py

```diff
@@ -785,10 +792,11 @@ def download_chunks_concurrently(
     performance under normal circumstances due to Python interpreter threading
     behavior. The default is therefore to use processes instead of threads.
 
+    :type bucket: :class:`google.cloud.storage.bucket.Bucket`
     :param bucket:
         The bucket which contains the blobs to be downloaded
```

Comment:
Seems like no bucket param for this method

Reply:
Wow, you're right. Good catch!!